packages/amplify-provider-awscloudformation/src/initializer.ts

```diff
@@ -1,29 +1,36 @@
+/* eslint-disable import/no-extraneous-dependencies */
```

Comment:
This shouldn't be ignored. If there's a dependency not present in the package.json it should be added to package.json